DOCS-1932: Review blink an led tutorial for R2D2 (#2736)
viamrobotics/docs: docs/tutorials/get-started/blink-an-led.md

> Before proceeding, be sure that you have [connected your Pi to the Viam app](/get-started/installation/#install-viam-server).

Reviewer: ```suggestion
Before proceeding, be sure that you have [connected your Pi to the Viam app](#project-setup).
```
[opt] So that they know it's instructions they've already completed if they've been following along

Now it's time to configure your machine's components.

> First, go to the [Viam app](https://app.viam.com/) and navigate to your new machine's **CONFIGURE** tab.

Reviewer: ```suggestion
Go to the [Viam app](https://app.viam.com/) and navigate to your new machine's **CONFIGURE** tab.
```
[opt] I know it's out-of-diff but I think "first" should come out since the proceeding paragraph says "before"... and we don't actually say "next" at any point

{{< tabs >}}
{{% tab name="Config Builder" %}}

Add a [_board component_](/components/board/) to represent your single-board computer, which in this case is the Raspberry Pi.
To create the new component, click the **+** icon next to your machine {{< glossary_tooltip term_id="part" text="part" >}} in the left-hand menu and select **Component**.
Select the `board` type, then select the `pi` model.
Enter a name for your board and click **Create**.
We used the name `"local"`.

Your board component panel will look like this:

> {{<imgproc src="/tutorials/blink-an-led/board-config.png" resize="1200x" style="width: 650px" alt="Config tab of the Viam app showing the board configuration. The board is named 'local' and the attributes are shown as empty braces.">}}

Reviewer: ```suggestion
{{<imgproc src="/tutorials/blink-an-led/board-config.png" resize="1200x" style="width: 650px" alt="Config tab of the Viam app showing the board configuration. The board is named 'local' and no attributes are configured.">}}
```

Author: Great catch!
